x-pack/plugin/sql/qa/src/main/resources/date.csv-spec

```diff
@@ -0,0 +1,89 @@
+//
+// Date
+//
+
+dateExtractDateParts
+SELECT
+DAY(CAST(birth_date AS DATE)) d,
+DAY_OF_MONTH(CAST(birth_date AS DATE)) dm,
+DAY_OF_WEEK(CAST(birth_date AS DATE)) dw,
+DAY_OF_YEAR(CAST(birth_date AS DATE)) dy,
+ISO_DAY_OF_WEEK(CAST(birth_date AS DATE)) iso_dw,
+WEEK(CAST(birth_date AS DATE)) w,
+QUARTER(CAST(hire_date AS DATE)) q,
```

Comment:
Here you used a different field... intentional or was just a small miss? (I was looking at the data returned by the query and the quarters didn't make any sense :-)), then I noticed this is a different field)

Reply:
it was copied from here: https://github.com/elastic/elasticsearch/blob/master/x-pack/plugin/sql/qa/src/main/resources/datetime.sql-spec#L35 
will change.